bookshelf/config/application.rb

```diff
@@ -13,15 +11,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+require_relative 'boot'
+
 require "rails"
 # Pick the frameworks you want:
 require "active_model/railtie"
-require "active_job/railtie"
```

Comment:
It looks like I was wrong about this one line. It seems to be required (but only in production mode). Please restore it.